### DISCUSSION TITLE
Allow viewing all aliases in one page.

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [anonaddy/anonaddy#509] [Feature Requests]
Not possible since rebrand to https://app.addy.io/aliases from https://app.anonaddy.com/aliases. I suggest dynamic viewing of the alias list, meaning that exactly the amount of aliases that shall fit the current display are loaded. Requested by https://github.com/anonaddy/anonaddy/issues/332#issuecomment-1692415568 and myself.

**Comment by willbrowningme** [comment]
Some users have tens of thousands of aliases so it is not feasible to show them all on one page. That is why the API and aliases page are paginated.

I've just added a new page size option to the aliases page so you can now view up to 100 per page.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/anonaddy/anonaddy/discussions/509#discussioncomment-6824609

@willbrowningme, do you mind making that an input form so that I can choose an arbitrary value, instead of a preset? It could have a maximum allowed.
